plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPaneltem.tsx

```diff
@@ -5,20 +5,35 @@ import './TopologyResourcesTabPanelItem.css';
 type TopologyResourcesTabPanelItemProps = {
   resourceLabel: string;
   dataTest?: string;
+  showResCount?: number;
 };
 
 const TopologyResourcesTabPanelItem = ({
   resourceLabel,
   children,
   dataTest,
+  showResCount,
 }: React.PropsWithChildren<TopologyResourcesTabPanelItemProps>) => {
   const emptyState = (
     <span className="bs-topology-text-muted">{`No ${resourceLabel} found for this resource.`}</span>
   );
   return (
     <>
-      <h2 className="topology-resources-tab-item-title">{resourceLabel}</h2>
-      <ul className="topology-resources-tab-item-list" data-testid={dataTest}>
+      <div className="bs-topology-resources-tab-item-title">
+        <h2 className="bs-topology-resources-tab-item-label">
+          {resourceLabel}
+        </h2>
+        {showResCount ? (
+          <span
+            className="bs-topology-text-muted"
+            data-testid="res-show-count"
+          >{`Showing latest ${showResCount} ${resourceLabel}`}</span>
```

Comment:
There are more than 3 PipelineRuns, the side panel does shows the latest 3, but the muted text is missing 
 

<img width="810" alt="Screenshot 2023-08-01 at 8 03 02 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/e965ed7e-aeee-43cb-969c-4189e6899334">